source/index.html

```diff
@@ -248,3 +248,11 @@ <h1>Home Assistant in the press</h1>
     </div>
   </div>
 </div>
+<script
+  src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+  integrity="sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs="
+  crossorigin="anonymous"></script>
+<script type="text/javascript">
+jQuery( "ul.menu li:has(ul)" ).addClass( "menuhasdropdown" );
+</script>
```

Comment:
I think using javascript for this is a bit overkill...

@ludeeus Could you help out with a review or hint here? 🙏

Reply:
Will take a look tomorrow 👍, even if this has to be done with JavaScript, we should use a regular query selector instead of importing jQuery

Reply:
<https://github.com/home-assistant/home-assistant.io/pull/19052> Alternative using CSS only, but personally I like that the menu items drop a little down on hover.

Reply:
Another option is just manually adding a class for li items in the html menu choices with nested items in [this file](https://github.com/home-assistant/home-assistant.io/blob/2a083a863369eee40480b91a75ee7141a616a453/source/_includes/site/header.html)

Reply:
> Alternative using CSS only, but personally I like that the menu items drop a little down on hover.

Agreed. So maybe this is the way to go then?

Reply:
> Another option is just manually adding a class for li items in the html menu choices with nested items in this file

You can do that with CSS selectors in the scss files 

> Agreed. So maybe this is the way to go then?

What I meant is, I like the current behavior (the current live site) :smile:

Reply:
Right, ok. No specific opinion on that myself :)

Reply:
> > Another option is just manually adding a class for li items in the html menu choices with nested items in this file
> 
> You can do that with CSS selectors in the scss files

I mean, if you add a class 'menuhasdropdown' to the `<li>` in the following 

```
<li>
            <a href="/docs/">Documentation <i class="icon icon-caret-down"></i></a>
.....
```

So a change to 

```
<li class="menuhasdropdown">
            <a href="/docs/">Documentation <i class="icon icon-caret-down"></i></a>
.....
```

Then it would work with the CSS changes in this PR, without the use of any javascript. The only thing is that you have to think of adding this class each time a new navigation item is added in the future.